plasmapy/physics/tests/test_transport.py

```diff
@@ -421,6 +434,9 @@ def test_fail1(self):
                   f"should not be equal to {fail1}.")
         assert testTrue, errStr
 
+    with pytest.raises(u.UnitConversionError):
```

Comment:
The tests you removed in 5e1509aed67608663ae7c22debc405886f3cf000 originated here, somehow. Messy rebase?

Reply:
Yea I think the rebase was sloppy. I clearly still have some learning to do on how to properly rebase.